packages/@biomejs/backend-jsonrpc/src/workspace.ts

```diff
@@ -74,7 +74,7 @@ export interface FilesConfiguration {
 	 */
 	ignoreUnknown?: boolean;
 	/**
-	 * The maximum allowed size for source code files in bytes. Files above this limit will be ignored for performance reason. Defaults to 1 MiB
+	 * The maximum allowed size for source code files in bytes. Files above this limit will be ignored for performance reasons. Defaults to 1 MiB
```

Comment:
I have no idea why this change was requested.

Reply:
It's possible that your fork is not up to date with `upstream`